### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by anna-lybid**

### CODE REVIEW

File: app/managers.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        self._connection = sqlite3.connect("cinema.sqlite")
+
+        with self._connection:
+            self._connection.execute(
+                f"CREATE TABLE IF NOT EXISTS {self.table_name} ("
+                "id INTEGER PRIMARY KEY, "
+                "first_name VARCHAR(50) NOT NULL, "
+                "last_name VARCHAR(50) NOT NULL"
+                ")"
+            )
```
Comment (ORENYT) [inline app/managers.py:18]: I thought we have to create the table from DB sql window.

File: app/managers.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            f"INSERT INTO {self.table_name} "
+            f"(first_name, last_name) VALUES (?, ?)",
+            (first_name, last_name))
+        self._connection.commit()
+
+    def all(self) -> list[Actor]:
+        actors = self._connection.execute(
+            f"SELECT * FROM {self.table_name}"
+        )
+        return [Actor(*row) for row in actors]
```
Comment (ORENYT) [inline app/managers.py:32]: good idea to use unpacking

**Review by ORENYT — APPROVED** [review]
Nice solution

File: app/managers.py (hunk 1 above)
Comment (Kroll410) [reply app/managers.py:18]: @ORENYT  It wasn't mentioned

**Pull request closed without merging**